apps/bfd-model/bfd-model-rif-samples/src/main/java/gov/cms/bfd/model/rif/samples/StaticRifFile.java

```diff
@@ -10,6 +10,7 @@
 
 /** This {@link RifFile} implementation operates on local files. */
 final class StaticRifFile implements RifFile {
+  /** Static Rif Resource. */
```

Comment:
Couldn't we do something more here; for example:
Enum {@link StaticRifResource} that identifies a static RIF file, its type, and expected record count.

Reply:
done